### ISSUE TITLE
twitterscraper - collect tweets without specifying word

### ISSUE THREAD

**Issue by clpalomi** [JustAnotherArchivist/snscrape#990]
Is it possible to use TwitterSearchScraper to collect a number of tweets just setting parameters like tweet.lang or tweet.place.country, without specifying a word as an argument?

**Comment by JustAnotherArchivist** [comment]
Yes, e.g. `lang:en` searches for English tweets (according to Twitter's detection, which isn't always accurate). Some filters can't be used on their own; adding `(filter:safe OR -filter:safe)` is one possible workaround for that. Refer to https://github.com/igorbrigadir/twitter-advanced-search for available filters (but some of these don't work anymore).
